### PR TITLE
Update the text to fit in the module change; provide usage examples

### PR DESCRIPTION
**Pull request by QiufangMa**
#38 #39

### CODE REVIEW

File: draft-ma-opsawg-schedule-yang.md [hunk 1]
```diff
@@ -203,22 +267,23 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
 --- back
 
-# Examples
+# Examples of Format Representation {#usage}
 
-   The following subsections provide some examples to illustrate the use of the period and recurrence formats defined as
-   YANG groupings. Only the message body is provided with JSON used for encoding {{?RFC7951}}.
+   This section provides some examples to illustrate the use of the
+   period and recurrence formats defined as YANG groupings. Note that "grouping"
+   does not define any nodes in the schema tree, the examples illustrated are
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:274]:
```suggestion
   does not define any data nodes in the schema tree, the examples illustrated are
```

File: draft-ma-opsawg-schedule-yang.md [hunk 2]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+{
+  "period-start": "2023-11-19T02:00:00",
+  "time-zone-identifier": "America/Los_Angeles",
+  "duration": "P20W"
+}
+~~~~
+
+## The "recurrence" Grouping
+
+   The following snippet can be used to indicate a every other day recurrence for
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:313]:
```suggestion
   The following snippet can be used to indicate a daily recurrence for
```

File: draft-ma-opsawg-schedule-yang.md [hunk 3]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+  "frequency": "ietf-schedule:yearly",
+  "until": "2050-11-25"
 }
 ~~~~
 
-## Recurrence Rule
+## The "recurrence-with-date-times" Grouping
 
-  The following snippet can be used to indicate a daily recurrent in December:
+   The following snippet can be used to indicate a recurrence that occur every 2
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:342]:
```suggestion
   The following snippet can be used to indicate a recurrence that occurs every 2
```

File: draft-ma-opsawg-schedule-yang.md [hunk 4]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+  },
+  "frequency": "ietf-schedule:hourly",
+  "interval": 2,
+  "until": "2023-12-01T17:00:00Z",
+  "date-times": ["2023-12-01T18:00:00Z"]
 }
 ~~~~
 
-   The following snippet can be used to indicate 10 occurrences that occur every last Saturday of the month:
+   The following snippet can be used to indicate a recurrence that occur every
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:357]:
```suggestion
   The following snippet can be used to indicate a recurrence that occurs every
```

File: draft-ma-opsawg-schedule-yang.md [hunk 5]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+      "duration": "PT00:00:20"
+    }
+   ]
 }
 ~~~~
 
-   The following indicates the example of a recurrence that occurs on the last workday of the month until December 25, 2023:
+## The "icalendar-recurrence" Grouping
+
+   The following snippet can be used to indicate 10 occurrences that occur at
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:385]:
```suggestion
   The following snippet indicates 10 occurrences that occur at
```

File: draft-ma-opsawg-schedule-yang.md [hunk 6]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+  "byday": [
+    {
+      "direction": [-1],
+      "weekday": "saturday"
+    }
+  ]
 }
 ~~~~
 
+   The following indicates the example of a recurrence that occurs on the last
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:405]:
```suggestion
   The following is an example of a recurrence that occurs on the last
```

File: draft-ma-opsawg-schedule-yang.md [hunk 7]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
-    "freq": "weekly",
-    "interval": 2,
-    "byday": [
-      { "weekday": "tuesday" },
-      { "weekday": "sunday" }
-    ],
-    "wkst": "monday"
-  }
+  "recurrence-first": {
+  "date-time-start": "2023-12-01T09:00:00Z"
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:433]:
```suggestion
    "date-time-start": "2023-12-01T09:00:00Z"
```

File: yang/example-scheduled-backup.yang [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+  container scheduled-backup-tasks {
+    description
+      "A container for backing up all current running configuration
+       on the device.";
+
+    choice local-or-remote {
+      description
+        "Specify whether the configuration to be backed up is local or
```
Comment (boucadair) [inline yang/example-scheduled-backup.yang:37]:
```suggestion
        "Specifies whether the configuration to be backed up is local or
```

File: yang/example-scheduled-backup.yang [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+         remote.";
+      case local {
+        description
+          "Configuration parameter for backing up of local device.";
```
Comment (boucadair) [inline yang/example-scheduled-backup.yang:41]:
```suggestion
          "Configuration parameters for backing up of local device.";
```

File: yang/example-scheduled-backup.yang [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+        leaf local {
+          type empty;
+          description
+            "The parameter specifies the configuration to be backed up
+             is on the local device.";
+        }
+      }
+      case remote {
+        description
+          "Configuration parameter for backing up of remote device.";
```
Comment (boucadair) [inline yang/example-scheduled-backup.yang:51]:
```suggestion
          "Configuration parameters for backing up of remote devices.";
```

File: yang/example-scheduled-link-bandwidth.yang [hunk 11]
```diff
@@ -0,0 +1,97 @@
+module example-scheduled-link-bandwidth {
+  yang-version 1.1;
+  namespace "http://example.com/example-scheduled-link-bandwidth";
+  prefix "exscattr";
```
Comment (boucadair) [inline yang/example-scheduled-link-bandwidth.yang:4]:
```suggestion
  prefix "ex-scattr";
```

File: yang/example-scheduled-link-bandwidth.yang [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+
+  import ietf-network {
+    prefix "nw";
+    reference
+      "RFC 8345: A YANG Data Model for Network Topologies";
+
```
Comment (boucadair) [inline yang/example-scheduled-link-bandwidth.yang:10]:
```suggestion
```

File: yang/example-scheduled-link-bandwidth.yang [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+  }
+
+  organization
+    "Example, Inc.";
+
+  contact
+    "Support at example.com";
+
+  description
+    "Example of a module defining an scheduled link bandwidth.";
```
Comment (boucadair) [inline yang/example-scheduled-link-bandwidth.yang:26]:
```suggestion
    "Example of a module defining a scheduled link bandwidth.";
```

File: yang/example-scheduled-link-bandwidth.yang [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+      type uint64;
+      units "Kbps";
+      description
+        "Bandwidth values, expressed in kilobits per second.";
+    }
+  }
+
+  container link-attributes {
+    description
+      "The container for the link attributes definition.";
```
Comment (boucadair) [inline yang/example-scheduled-link-bandwidth.yang:48]:
```suggestion
      "Definition of link attributes.";
```

File: yang/example-scheduled-link-bandwidth.yang [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+    list link {
+      key "source-node destination-node";
+      description
+        "The list of the scheduled link attributes.";
+      leaf source-node {
+        type nw:node-id;
+        description
+          "The source node id.";
```
Comment (boucadair) [inline yang/example-scheduled-link-bandwidth.yang:56]:
```suggestion
          "Indicates the source node identifier.";
```

File: yang/example-scheduled-link-bandwidth.yang [hunk 16]
```diff
@@ -0,0 +1,97 @@
...
+      }
+      leaf destination-node {
+        type nw:node-id;
+        description
+          "The destination node id.";
```
Comment (boucadair) [inline yang/example-scheduled-link-bandwidth.yang:61]:
```suggestion
          "Indicates the destination node identifier.";
```

File: yang/example-scheduled-link-bandwidth.yang [hunk 17]
```diff
@@ -0,0 +1,97 @@
...
+      leaf default-bandwidth {
+        type uint64;
+        units "Kbps";
+        description
+          "Default bandwidth values when unspecified.";
+      }
+
+      choice time-variant-type {
+        description
+          "The schedule type.";
```
Comment (boucadair) [inline yang/example-scheduled-link-bandwidth.yang:73]:
```suggestion
          "Controls the schedule type.";
```

File: yang/example-scheduled-link-bandwidth.yang [hunk 18]
```diff
@@ -0,0 +1,97 @@
...
+        case period {
+          uses schedule:period-of-time;
+        }
+        case recurrence {
+          uses schedule:recurrence-with-date-times {
+            augment "date-times-choice/period-timeticks/period-timeticks" {
+              description
+                "Allow the attributes to be specified inside each period
```
Comment (boucadair) [inline yang/example-scheduled-link-bandwidth.yang:81]:
```suggestion
                "Specifies the attributes inside each period
```

File: yang/example-scheduled-link-bandwidth.yang [hunk 19]
```diff
@@ -0,0 +1,97 @@
...
+                 -timesticks entry.";
+              uses link-bandwidth-grouping;
+            }
+            augment "date-times-choice/period/period" {
+              description
+                "Allow the attributes to be specified inside each period
```
Comment (boucadair) [inline yang/example-scheduled-link-bandwidth.yang:87]:
```suggestion
                "Specifies the attributes within each period
```

File: draft-ma-opsawg-schedule-yang.md [hunk 20]
```diff
@@ -203,22 +267,23 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+   This section provides some examples to illustrate the use of the
+   period and recurrence formats defined as YANG groupings. Note that "grouping"
+   does not define any nodes in the schema tree, the examples illustrated are
+   just for the ease of understanding. Only the message body is provided with
+   JSON used for encoding {{?RFC7951}}.
 
-## Period of Time
+## The "period-of-time" Grouping
 
    The example of a period that starts at 08:00:00 UTC, on January 1, 2023 and ends at 18:00:00 UTC
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:280]:
```suggestion
   The example of a period that starts at 08:00:00 UTC, on January 1, 2025 and ends at 18:00:00 UTC
```

File: draft-ma-opsawg-schedule-yang.md [hunk 21]
```diff
@@ -203,22 +267,23 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
    on December 31, 2025 is encoded as follows:
 
 ~~~~
 {
-  "period-of-time": {
-    "period-start": "2023-01-01T08:00:00Z",
-    "period-end": "2025-12-01T18:00:00Z"
-  }
+  "period-start": "2023-01-01T08:00:00Z",
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:285]:
```suggestion
  "period-start": "2025-01-01T08:00:00Z",
```

File: draft-ma-opsawg-schedule-yang.md [hunk 22]
```diff
@@ -203,22 +267,23 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+  "period-end": "2025-12-01T18:00:00Z"
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:286]:
```suggestion
  "period-end": "2027-12-01T18:00:00Z"
```

File: draft-ma-opsawg-schedule-yang.md [hunk 23]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
 
 ~~~~
 {
-  "period-of-time": {
-    "period-start": "2023-01-01T08:00:00Z",
-    "period-duration": "P15DT05:20:00"
-  }
+  "period-start": "2023-01-01T08:00:00Z",
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:295]:
```suggestion
  "period-start": "2025-01-01T08:00:00Z",
```

File: draft-ma-opsawg-schedule-yang.md [hunk 24]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+  "duration": "P15DT05:20:00"
+}
+~~~~
+
+   An example of a period that starts at 2:00 A.M. in Los Angeles on November 19,
+   2023 and lasts 20 weeks:
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:301]:
```suggestion
   2025 and lasts 20 weeks:
```

File: draft-ma-opsawg-schedule-yang.md [hunk 25]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+
+~~~~
+{
+  "period-start": "2023-11-19T02:00:00",
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:305]:
```suggestion
  "period-start": "2025-11-19T02:00:00",
```

File: draft-ma-opsawg-schedule-yang.md [hunk 26]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+  "time-zone-identifier": "America/Los_Angeles",
+  "duration": "P20W"
+}
+~~~~
+
+## The "recurrence" Grouping
+
+   The following snippet can be used to indicate a every other day recurrence for
+   10 occurrences, starting at 3 p.m. on December 1, 2023 in the Eastern United States time zone:
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:314]:
```suggestion
   10 occurrences, starting at 3 p.m. on December 1, 2025 in the Eastern United States time zone:
```

File: draft-ma-opsawg-schedule-yang.md [hunk 27]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+
+~~~~
+{
+  "recurrence-first": {
+    "date-time-start": "2023-11-01T15:00:00",
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:319]:
```suggestion
    "date-time-start": "2025-11-01T15:00:00",
```

File: draft-ma-opsawg-schedule-yang.md [hunk 28]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+  "until": "2050-11-25"
 }
 ~~~~
 
-## Recurrence Rule
+## The "recurrence-with-date-times" Grouping
 
-  The following snippet can be used to indicate a daily recurrent in December:
+   The following snippet can be used to indicate a recurrence that occur every 2
+   hours from 9:00 AM to 5:00 PM, and 6PM UTC time on 2023-12-01:
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:343]:
```suggestion
   hours from 9:00 AM to 5:00 PM, and 6PM UTC time on 2025-12-01:
```

File: draft-ma-opsawg-schedule-yang.md [hunk 29]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
 
 ~~~~
 {
-  "recurrence": {
-    "freq": "daily",
-    "byyearmonth": [12]
-  }
+  "recurrence-first": {
+    "date-time-start": "2023-12-01T09:00:00Z"
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:348]:
```suggestion
    "date-time-start": "2025-12-01T09:00:00Z"
```

File: draft-ma-opsawg-schedule-yang.md [hunk 30]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+  },
+  "frequency": "ietf-schedule:hourly",
+  "interval": 2,
+  "until": "2023-12-01T17:00:00Z",
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:352]:
```suggestion
  "until": "2025-12-01T17:00:00Z",
```

File: draft-ma-opsawg-schedule-yang.md [hunk 31]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+  "date-times": ["2023-12-01T18:00:00Z"]
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:353]:
```suggestion
  "date-times": ["2025-12-01T18:00:00Z"]
```

File: draft-ma-opsawg-schedule-yang.md [hunk 32]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
 }
 ~~~~
 
-   The following snippet can be used to indicate 10 occurrences that occur every last Saturday of the month:
+   The following snippet can be used to indicate a recurrence that occur every
+   30 minutes and last for 15 minutes from 9:00 AM to 5:00 PM, and extra two occurrences
+   at 6:00 PM and 6:30 PM with each lasting for 20 minutes on 2023-12-01:
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:359]:
```suggestion
   at 6:00 PM and 6:30 PM with each lasting for 20 minutes on 2025-12-01:
```

File: draft-ma-opsawg-schedule-yang.md [hunk 33]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
-    "count": 10,
-    "byday": [
-      {
-        "direction": [-1],
-        "weekday": "saturday"
-      }
-    ]
-  }
+  "recurrence-first": {
+    "date-time-start": "2023-12-01T09:00:00Z",
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:364]:
```suggestion
    "date-time-start": "2025-12-01T09:00:00Z",
```

File: draft-ma-opsawg-schedule-yang.md [hunk 34]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+    "duration": "PT00:00:15"
+  },
+  "frequency": "ietf-schedule:minutely",
+  "interval": 30,
+  "until": "2023-12-01T17:00:00Z",
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:369]:
```suggestion
  "until": "2025-12-01T17:00:00Z",
```

File: draft-ma-opsawg-schedule-yang.md [hunk 35]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+  "period": [
+    {
+      "period-start": "2023-12-01T18:00:00Z",
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:372]:
```suggestion
      "period-start": "2025-12-01T18:00:00Z",
```

File: draft-ma-opsawg-schedule-yang.md [hunk 36]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+      "duration": "PT00:00:20"
+    },
+    {
+      "period-start": "2023-12-01T18:30:00Z",
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:376]:
```suggestion
      "period-start": "2025-12-01T18:30:00Z",
```

File: draft-ma-opsawg-schedule-yang.md [hunk 37]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+    { "weekday": "friday"}
+  ],
+  "bysetpos": [-1]
+}
+~~~~
 
-   The following depicts the example of a recurrence that occurs every other week on Tuesday and Sunday, the week starts from Monday:
+  The following snippet can be used to indicate a recurrence that occur every 20
+  minutes from 9:00 AM to 4:40 PM (UTC), with the occurrence starting at 10:20 AM
+  being excluded on 2023-12-01:
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:428]:
```suggestion
  being excluded on 2025-12-01:
```

File: draft-ma-opsawg-schedule-yang.md (hunk 7 above)
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:433]:
```suggestion
  "date-time-start": "2025-12-01T09:00:00Z"
```

File: draft-ma-opsawg-schedule-yang.md [hunk 38]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
-    "byday": [
-      { "weekday": "tuesday" },
-      { "weekday": "sunday" }
-    ],
-    "wkst": "monday"
-  }
+  "recurrence-first": {
+  "date-time-start": "2023-12-01T09:00:00Z"
+  },
+  "until": "2023-12-01T16:40:00Z",
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:435]:
```suggestion
  "until": "2025-12-01T16:40:00Z",
```

File: draft-ma-opsawg-schedule-yang.md [hunk 39]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+  "frequency": "ietf-schedule:minutely",
+  "byminute": [0, 20, 40],
+  "byhour": [9, 10, 11, 12, 13, 14, 15, 16],
+  "exception-dates": ["2023-12-01T10:20:00Z"]
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:439]:
```suggestion
  "exception-dates": ["2025-12-01T10:20:00Z"]
```

**Review by boucadair — CHANGES_REQUESTED** [review]
Thanks. 

The dates in the examples should be in general in the future taking into account the publication of the doc as RFC.

File: draft-ma-opsawg-schedule-yang.md [hunk 40]
```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+~~~~
+
+  Following is a configuration example which indicates a link's bandwidth is
+  scheduled between 2023/12/01 0:00 UTC to the end of 2023/12/31 with a daily
+  schedule. In each day, the bandwidth value is scheduled to be 500Kbps between
+  1:00 AM to 6:00 AM and 800Kbps between 10:00 PM to 11:00 PM. The bandwidth
+  value that's not covered by the period above is 1000Kbps by default.
+
+~~~~
+{::include ./examples/example-scheduled-link-bandwidth.xml}
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:478]:
```suggestion
{::include-fold ./examples/example-scheduled-link-bandwidth.xml}
```

File: draft-ma-opsawg-schedule-yang.md [hunk 41]
```diff
@@ -99,32 +99,75 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+   Each of these groupings are presented in the following subsections. Examples
+   are provided in {{usage}}.
+
+## The "period-of-time" Grouping
+
+   The "period-of-time" grouping allows a time period to be represented using
+   either a start ("period-start") and end date and time ("period-end"), or a
+   start ("period-start") and a positive time duration ("duration"). For the first
+   format, the start of the period MUST be before the end of the period
+   {{Section 3.3.9 of ?RFC5545}}.
```
[the remark this answers is not in the document]
Comment (QiufangMa) [reply draft-ma-opsawg-schedule-yang.md:126]: I am unsure about this, should it be a normative reference?

[the remark this answers is not in the document]
Comment (boucadair) [reply draft-ma-opsawg-schedule-yang.md:126]: Yes as many rules are defined there. Thanks.

File: draft-ma-opsawg-schedule-yang.md [hunk 42]
```diff
@@ -99,32 +99,75 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+
+## The "recurrence" Grouping
+
+  The "recurrence" grouping specifies a simple recurrence rule, the definition
+  conforms to part of the "recurrence rule" properties in {{Section 3.3.10 of ?RFC5545}}.
```
[the remark this answers is not in the document]
Comment (boucadair) [reply draft-ma-opsawg-schedule-yang.md:131]: Same reply, then :-)

File: draft-ma-opsawg-schedule-yang.md (hunk 41 above)
[the remark this answers is not in the document]
Comment (QiufangMa) [reply draft-ma-opsawg-schedule-yang.md:126]: ok.

File: draft-ma-opsawg-schedule-yang.md (hunk 2 above)
Comment (QiufangMa) [reply draft-ma-opsawg-schedule-yang.md:313]: There is a difference between "every other day" and "daily", by "daily", the interval value is 1, while by "every other day", we mean “every two days”, the interval value is 2.

File: yang/example-scheduled-backup.yang (hunk 9 above)
Comment (QiufangMa) [reply yang/example-scheduled-backup.yang:41]: Plural form? We only defined one parameter under this case.

Comment (boucadair) [reply yang/example-scheduled-backup.yang]: We need to distinguish the intent of the container vs the current parameter'(s) that are listed out there.

**Comment by QiufangMa** [comment]
> Thanks.
> 
> The dates in the examples should be in general in the future taking into account the publication of the doc as RFC.

Thanks Med, all fixed.

**Pull request merged**